# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/tests/test_dom_input_event_on_htmleditor.html
# Commit: 366245552cae
# Full Hash: 366245552cae536ef883be806322395dc47b759d
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-06-17 09:24:05
# Regressor Bug: 1951038
# File Overlap Count: 1
# Description:
#   Bug 1951038 - Get rid of the legacy white-space normalizer of `HTMLEditor` and the pref r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D253589
# ==============================================================================

diff -r b13b0a530402 -r 366245552cae editor/libeditor/tests/test_dom_input_event_on_htmleditor.html
--- a/editor/libeditor/tests/test_dom_input_event_on_htmleditor.html	Tue Jun 17 00:10:27 2025 +0000
+++ b/editor/libeditor/tests/test_dom_input_event_on_htmleditor.html	Tue Jun 17 00:37:22 2025 +0000
@@ -32,8 +32,6 @@
 function runTests() {
   const kWordSelectEatSpaceToNextWord =
     SpecialPowers.getBoolPref("layout.word_select.eat_space_to_next_word");
-  const kNewWhiteSpaceNormalizerEnabled =
-    SpecialPowers.getBoolPref("editor.white_space_normalization.blink_compatible");
 
   const kImgURL =
     "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAYAAABytg0kAAAAEElEQVR42mNgaGD4D8YwBgAw9AX9Y9zBwwAAAABJRU5ErkJggg==";
@@ -870,7 +868,7 @@
           SpecialPowers.doCommand(aWindow, "cmd_deleteWordBackward");
         },
         {
-          innerHTML: kNewWhiteSpaceNormalizerEnabled ? "abc&nbsp;" : "abc <br>",
+          innerHTML: "abc&nbsp;",
           beforeInputEvent: {
             cancelable: true,
             inputType: "deleteWordBackward",
@@ -912,12 +910,7 @@
           SpecialPowers.doCommand(aWindow, "cmd_deleteWordForward");
         },
         {
-          innerHTML: (function () {
-            if (kWordSelectEatSpaceToNextWord) {
-              return "def";
-            }
-            return kNewWhiteSpaceNormalizerEnabled ? "&nbsp;def" : " def"
-          })(),
+          innerHTML: kWordSelectEatSpaceToNextWord ? "def" : "&nbsp;def",
           beforeInputEvent: {
             cancelable: true,
             inputType: "deleteWordForward",